cypress/integration/registry/newUserRegistration.js

```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
+const randomUser = require('../../helpers/randomUser')
+
+describe('Register new user', async function () {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  //Register new user with all valid data
+  it('New user registers and verifies primary email', function () {
+    // Bypass the duplicated research call to avoid getting the "Is this you modal"
+    cy.intercept('GET', Cypress.env('duplicatedModalEndPoint'), [])
+    //generate a new (random) user
+    const userToRegister = randomUser()
+
+    cy.get('#menu-signin-button').click()
+    //verify user is redirected to the sign in page
+    cy.url().should('include', '/signin')
+    cy.get('#register-button').click()
+    //verify user is redirected to the register page
+    cy.url().should('include', '/register')
+
+    //STEP 1/3 populate form
+    cy.get('#given-names-input').clear().type(userToRegister.name)
+    cy.get('#family-names-input').clear().type(userToRegister.familyName)
+    cy.get('#email-input').clear().type(userToRegister.email)
+    cy.get('#confirm-email-input').clear().type(userToRegister.email)
+    cy.get('#step-a-next-button').click()
+
+    //STEP 2/3 create OID
+    cy.get('.mat-card-title').contains('Create your ORCID iD')
+    cy.get('#password-input').clear().type(userToRegister.password)
+    cy.get('#password-confirm-input').clear().type(userToRegister.password)
+    cy.get('.ng-valid #step-b-next').click()
+
+    //STEP 3/3
+    cy.get('#visibility-everyone-input-input').click({ force: true })
+    cy.get('#privacy-input-input').check({ force: true }).should('be.checked')
+    cy.get('#data-processed-input-input')
+      .check({ force: true })
+      .should('be.checked')
+
+    //CAPTCHA
+    // Wrap iframe body into a cypress object and perform test within there
+    cy.getIframeBody('iframe').within(() => {
+      cy.get('.recaptcha-checkbox-border').click()
+      cy.get('#recaptcha-anchor', { timeout: 10000 }).should(
+        'have.class',
+        'recaptcha-checkbox-checked'
+      )
+    })
+    cy.get('#step-c-register-button').click()
+    cy.wait(10000) //wait 10 secs before checking for the email
```

Comment:
Having a way time of 10seconds is never a great thing!. And the emails might get delayed for more than 10 seconds sometimes. 

Checkin on docs of the library https://www.npmjs.com/package/gmail-tester-extended

It seems like we can later use the `check_inbox` function. Wich will automatically check for the email until it shows up in the inbox. 


(We can leave this change for later)

Reply:
added a TO DO, will fix it for next round